src/platform/elements/loading/Loading.ts

```diff
@@ -102,3 +102,31 @@ export class NovoSpinnerElement {
     @Input() inverse: boolean;
     @Input() baseHref: string;
 }
+
+
+@Component({
```

Comment:
why does the other spinner not work?

Reply:
the simple-spinner is smaller and more fitting for the showing the loading state within the picker. ALso, this was the UX preference